src/main/java/frc/trigon/robot/subsystems/ampaligner/AmpAlignerConstants.java

```diff
@@ -0,0 +1,113 @@
+    private static final StaticFeedforwardSignValue STATIC_FEEDFORWARD_SIGN_VALUE = StaticFeedforwardSignValue.UseVelocitySign;
+    private static final double GEAR_RATIO = 52;
+    static final boolean FOC_ENABLED = true;
+    static final Rotation2d LIMIT_SWITCH_PRESSED_ANGLE = Rotation2d.fromDegrees(24);
+    static final double LIMIT_SWITCH_REPEAT_TIME_THRESHOLD_SECONDS = 0.1;
+
+    private static final int MOTOR_AMOUNT = 1;
+    private static final DCMotor GEARBOX = DCMotor.getFalcon500Foc(MOTOR_AMOUNT);
+    private static final double AMP_ALIGNER_MASS_KILOGRAMS = 1.1;
+    public static final double AMP_ALIGNER_LENGTH_METERS = 0.52;
+    private static final Rotation2d
+            AMP_ALIGNER_MINIMUM_ANGLE = LIMIT_SWITCH_PRESSED_ANGLE,
+            AMP_ALIGNER_MAXIMUM_ANGLE = Rotation2d.fromDegrees(180);
+    private static final SingleJointedArmSimulation SIMULATION = new SingleJointedArmSimulation(
+            GEARBOX,
+            GEAR_RATIO,
+            AMP_ALIGNER_LENGTH_METERS,
+            AMP_ALIGNER_MASS_KILOGRAMS,
+            AMP_ALIGNER_MINIMUM_ANGLE,
+            AMP_ALIGNER_MAXIMUM_ANGLE,
+            true
+    );
+
+    static final SysIdRoutine.Config SYSID_CONFIG = new SysIdRoutine.Config(
+            Units.Volts.of(2).per(Units.Second.of(1)),
+            Units.Volts.of(3),
+            Units.Second.of(1000)
+    );
+
+    static {
+        configureMotor();
+    }
+
+    private static void configureMotor() {
+        final TalonFXConfiguration config = new TalonFXConfiguration();
+
+        config.Audio.BeepOnBoot = false;
+        config.Audio.BeepOnConfig = false;
+
+        config.MotorOutput.Inverted = INVERTED_VALUE;
+        config.MotorOutput.NeutralMode = NEUTRAL_MODE_VALUE;
+
+        config.HardwareLimitSwitch.ReverseLimitType = REVERSE_LIMIT_TYPE_VALUE;
+        config.HardwareLimitSwitch.ReverseLimitSource = REVERSE_LIMIT_SOURCE_VALUE;
+
+        config.Slot0.kP = P;
+        config.Slot0.kI = I;
+        config.Slot0.kD = D;
+        config.Slot0.kS = KS;
+        config.Slot0.kV = KV;
+        config.Slot0.kA = KA;
+        if (RobotHardwareStats.isSimulation())
+            config.Slot0.kG = KG;
+
+        config.Slot0.GravityType = GRAVITY_TYPE_VALUE;
+        config.Slot0.StaticFeedforwardSign = STATIC_FEEDFORWARD_SIGN_VALUE;
+
+        config.Feedback.RotorToSensorRatio = GEAR_RATIO;
+
+        MOTOR.applyConfiguration(config);
+        MOTOR.setPhysicsSimulation(SIMULATION);
+
+        MOTOR.registerSignal(TalonFXSignal.POSITION, 100);
+        MOTOR.registerSignal(TalonFXSignal.VELOCITY, 100);
+        MOTOR.registerSignal(TalonFXSignal.MOTOR_VOLTAGE, 100);
+        MOTOR.registerSignal(TalonFXSignal.CLOSED_LOOP_REFERENCE, 100);
+        MOTOR.registerSignal(TalonFXSignal.REVERSE_LIMIT, 100);
+        MOTOR.registerSignal(TalonFXSignal.STATOR_CURRENT, 100);
+    }
+
+    public enum AmpAlignerState {
+        OPENED(AMP_ALIGNER_MAXIMUM_ANGLE),
+        CLOSED(AMP_ALIGNER_MINIMUM_ANGLE);
```

Comment:
CLOSE and OPEN 
Also, CLOSE should not be the maximum angle. It's a set one we'll decide.